### DISCUSSION TITLE
remove connection

### DISCUSSION THREAD

**Discussion by tianweimol** [elsa-workflows/elsa-core#976] [Q&A]
thanks for you answer
how can i remove a connect in designer?
is elsa designer support undo the previous step?

**Comment by sfmskywalker** [comment]
Thanks for reminding me to [document that.](https://elsa-workflows.github.io/elsa-core/docs/next/designer/designer-using-the-designer)

### Deleting Connections
To delete an existing connection, hover your mouse over it and then RIGHT-click your mouse.

### Undo / Redo
Undo/redo operations are currently not supported, but we have an issue for it.
